### PR TITLE
Add examples, integration tests and documentation

### PR DESCRIPTION
**Pull request by odsod**
- docs: add examples, integration tests and documentation to README
- chore: extract wkt package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,108 @@
...
+		project = "bigquery-public-data"
+		dataset = "san_francisco_film_locations"
+		table   = "film_locations"
+	)
+	// Connect to BigQuery.
+	client, err := bigquery.NewClient(ctx, project)
+	if err != nil {
+		panic(err) // TODO: Handle error.
+	}
+	// Load BigQuery rows into a library.Book message.
```
Comment (Jassob) [inline README.md:86]: Old comment?

```suggestion
	// Load BigQuery rows into a examplev1.FilmLocation message.
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,108 @@
...
+	messageLoader := &protobq.MessageLoader{
+		Message: &examplev1.FilmLocation{},
+	}
+	// Iterate rows in table.
+	rowIterator := client.Dataset(dataset).Table(table).Read(ctx)
+	for {
+		// Load next row into the library.Book message.
```
Comment (Jassob) [inline README.md:93]: And here

```suggestion
		// Load next row into the examplev1.FilmLocation message.
```

File: encoding/protobq/integration_test.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+}
+
+type publicDataSetIntegrationTest struct {
+	ProjectID        string
+	DatasetID        string
+	TableID          string
+	Limit            int
+	Message          proto.Message
+	MarshalOptions   protobq.MarshalOptions
```
Comment (Jassob) [inline encoding/protobq/integration_test.go:50]: These are unused in the test, I'd like to either add tests with different MarshalOptions or remove the field.

**Review by Jassob — COMMENTED** [review]
Looks really good to me, well done!

I found some outdated comments and have a comment about the integration tests:

File: encoding/protobq/integration_test.go (hunk 3 above)
Comment (odsod) [reply encoding/protobq/integration_test.go:50]: These are coming up in an upcoming PR - I have some unsubmitted material that I'm going to portion out after this PR!

File: README.md (hunk 2 above)
Comment (odsod) [reply README.md]: Nice find! Fixed

File: README.md (hunk 1 above)
Comment (odsod) [reply README.md]: Fixed!

**Review by Jassob — APPROVED** [review]
Great work, let's merge! :+1: :tada:

**Comment by odsod** [comment]
@Jassob Thanks! :raised_hands: Got some more stuff in the pipe.

**Pull request merged**